bootcamp-milestone-2/src/app/blog/page.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import BlogPreview from '../components/recipePreview';
+import { Blog } from '../components/blog'; // Import the Blog type
+import blogData from '../data/blogs.json'; // Import the JSON data
+import Header from '../components/header';
+import Layout from '../layout';
+import Styles from './blog.module.css';
+
+const Home: React.FC = () => {
+  const blogItems: Blog[] = blogData;
+  
+  return (
+    <Layout>
```

Comment:
Since you're just importing the root layout, you shouldn't need to add it to your pages as well. It's already applied globally. This applies to all your other pages too.

Reply:
This might be worth looking into for your run errors too.